app/js/components/labOrderEntry/styles.scss

```diff
@@ -143,4 +143,49 @@
     position: absolute;
     right: 10px;
   }
+
+  .action-btn-wrapper{
+    position: absolute;
+    top: 0;
+    right: 0;
+    display: flex;
+    width: 80px;
+  }
+
+  .action-btn {
+    cursor: pointer;
+    width: 50%;
+    height: 100% !important;
+    border-left: 1px #d0d0d0 solid;
+    display: inline-flex;
+    justify-content: center;
+    padding: .7em;
+    transition: all ease-in-out 200ms;
+    -moz-transition: all ease-in-out 200ms;
+    -webkit-transition: all ease-in-out 200ms;
+
+    #draft-discard-btn {
+      color: #ff3d3d;
+    }
+   }
+
+  .action-btn:hover {
+    background-color: rgba(0, 0, 0, .2);
+  }
+
+  .action-btn:hover > a > i.scale  {
+    text-decoration: none;
+    transform: scale(1.7);
+    -webkit-transform: scale(1.7);
+    -moz-transform: scale(1.7);
+  }
+
+  /* Icon styles */
```

Comment:
Hi @topseySuave.
Since we are using `SASS` for styling, do you mind nesting the three styles above to become something like this?
```
.action-btn: {
  // styles
  &:hover: {
    // styles
    & > a  > i.scale {
     // styles
    }
  }
}
```

Reply:
Done